vimeo-networking/src/main/java/com/vimeo/networking/model/Interaction.java

```diff
@@ -88,6 +110,10 @@ public String toString() {
     protected Stream mStream;
 
     @Nullable
+    @SerializedName("status")
+    protected IapStatus mStatus;
```

Comment:
Why protected?

Reply:
All other fields in this model are protected.  I actually have a jam branch to change *everything* to private w/ public accessors.  Going to keep with consistency here vs a big refactor as it's outside of scope here I think.